src/pyflex/window_selector.py

```diff
@@ -420,6 +452,8 @@ def reject_based_on_signal_to_noise_ratio(self):
         noise = self.observed.data[self.config.noise_start_index:
                                    self.config.noise_end_index]
 
+        offset = self.event.origin_time - self.observed.stats.starttime
```

Comment:
I think I am going to delete that...Cause I think it is not necessary to have this value. I am just going to set:
left = win.relative_starttime
right = win.relatvie_endtime